packages/example/utils/locale.ts

```diff
@@ -0,0 +1,51 @@
+import { Locale } from '@rainbow-me/rainbowkit';
+// Detect user locale from browser
+export const getDetectedBrowserLocale = () => {
+  if (typeof window !== 'undefined' && typeof navigator !== 'undefined') {
+    if (navigator.languages?.length) {
+      return navigator.languages[0];
+    }
+
+    if (navigator.language) {
+      return navigator.language;
+    }
+  }
+};
```

Comment:
I've done some additional checks in here to make sure it works with SSR too. Here you will either get array of languages which means we will pick the first one.

Otherwise if array of languages doesn't exist, but we have a single language (`navigator.language`) then we can choose that ✅

For older browsers like internet explorer this will return `undefined` if they don't support `navigator`.